File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        splited = s.split(" ")
        
        a_list = list(
            filter(
                lambda x: x != '', map(lambda x: x.strip(), splited)
            )
        )
        
        return len(a_list.pop())
            

File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution {
	public int lengthOfLastWord(String s) {
		List<String> strings = Arrays.stream(s.split(" "))
				.map(it -> it.trim())
				.filter(it -> it != "")
				.toList();

		String last = strings.get(strings.size() - 1);

		return last.length();
	}
}
